### PR TITLE
{bio}[foss/2022a] dorado v0.1.1 w/ CUDA 11.7.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires:
* #17194

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3303.joltik.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz (cascadelake), 1 x NVIDIA Tesla V100-SXM2-32GB, 525.60.13, Python 3.6.8
See https://gist.github.com/ecd89a75e5b30aee1bb854e90fbcc317 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/d/dorado/dorado-0.1.1-foss-2022a-CUDA-11.7.0.eb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+# don't link to OpenSSL static libraries
+# fix for CMake Error "missing: OPENSSL_CRYPTO_LIBRARY" (if only shared OpenSSL libraries are available)
+preconfigopts = "sed -i '/OPENSSL_USE_STATIC_LIBS TRUE/d' ../dorado/cmake/OpenSSL.cmake && "
+preconfigopts += "export OPENSSL_ROOT_DIR=$EBROOTOPENSSL && "
+
+# don't use vendored HTSlib, use provided HTSlib dependency
+preconfigopts += "sed -i '/add_dependencies.*htslib_project/d' ../dorado/CMakeLists.txt && "
+
+configopts = "-DDORADO_INSTALL_PATH=%(installdir)s "
+configopts +=  "-DCUDA_TOOLKIT_ROOT_DIR=$EBROOTCUDA -DCMAKE_CUDA_COMPILER=$EBROOTCUDA/bin/nvcc "
```
Comment (Micket) [inline easybuild/easyconfigs/d/dorado/dorado-0.1.1-foss-2022a-CUDA-11.7.0.eb:49]:
```suggestion
configopts += "-DCUDA_TOOLKIT_ROOT_DIR=$EBROOTCUDA -DCMAKE_CUDA_COMPILER=$EBROOTCUDA/bin/nvcc "
```

**Comment by benczaja** [comment]
Nice! Maybe I am mistaken, but isn't ('CMake', '3.23.1') part of the 2022a toolchain?

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in total)
bear-pg0103u14a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6330 CPU @ 2.00GHz (icelake), 1 x NVIDIA NVIDIA A30, 520.61.05, Python 3.6.8
See https://gist.github.com/5814509a0b42f584ecf272193632ea66 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0103u14a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6330 CPU @ 2.00GHz (icelake), 1 x NVIDIA NVIDIA A30, 520.61.05, Python 3.6.8
See https://gist.github.com/918ce44968322fe0837f7c8420d4beb4 for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**